pkgs/development/libraries/lib2geom/default.nix

```diff
@@ -37,17 +37,17 @@ stdenv.mkDerivation rec {
     gsl
     cairo
     double-conversion
-  ];
-
-  checkInputs = [
+    # Still needed even if doCheck is false
     gtest
   ];
 
   cmakeFlags = [
     "-D2GEOM_BUILD_SHARED=ON"
   ];
 
-  doCheck = true;
+  # Enable when this is fixed:
+  # https://gitlab.com/inkscape/lib2geom/-/issues/49
+  doCheck = !stdenv.hostPlatform.isi686;
```

Comment:
Can't we run the working tests?

Reply:
We probably can. The developers already fixed the issue upstream, so we even have a patch for it too.

It's just that I don't think we should touch any other platforms in any way for the sake of i686, and [it was you who contended on another MR](https://github.com/NixOS/nixpkgs/pull/200670#discussion_r1020809390) that conditionally applied patches aren't the best idea either.

Reply:
Right. But if we apply `https://gitlab.com/inkscape/lib2geom/-/commit/5b7c75dd3841cb415f163f0a81f556c57d3e0a83.patch` unconditionally, that should be fine.

Reply:
> It's just that I don't think we should touch any other platforms in any way for the sake of i686

It is totally fine to apply a patch for all platforms if it is only required for once.

Reply:
Reworked the MR, tested it again too.